scilpy/tractanalysis/bundle_operations.py

```diff
@@ -117,8 +118,7 @@ def uniformize_bundle_sft(sft, axis=None, ref_bundle=None, swap=False):
             else:
                 # Bitwise XOR
                 if (bool(labels[tuple(sft.streamlines[i][0].astype(int))] >
-                         labels[tuple(sft.streamlines[i][-1].astype(int))])
-                        ^ bool(swap)):
+                         labels[tuple(sft.streamlines[i][-1].astype(int))])):
```

Comment:
I trust you with the bitwise operations, but just want to make sure you didn't remove the `^ bool(swap)` part by accident?

Reply:
It's ok since the if at line 98 is swapping labels, this is checking orientation again based on that swap (so we don't want to cancel the previous swap)